scss/civicrm/contact/pages/_events.scss

```diff
@@ -1,3 +1,15 @@
+.page-civicrm-event-search {
+  [class*=CRM_Event_Form_Task_] {
+    .crm-form-block > .crm-submit-buttons > .crm-button:first-of-type {
+      margin-left: 0;
+    }
+
+    .status {
```

Comment:
why is the border removed only on this specific page?

Reply:
I have added a border to all status messages within the page, but Jon advised that for status messages that are inside the pane itself we should not have a white border. So that's why I put it here. Shall this be handled in another way?

Reply:
> I have added a border to all status messages within the page

Can you show me a screenshot of what status message you are talking about in this case?

> but Jon advised that for status messages that are inside the pane itself we should not have a border

then why aren't we doing this in a more generic way, instead of doing it on a page-by page basis?

Reply:
I have found this at this page, but as soon as it's loaded it's removed from here and made into an alert on the top right:
![message-outside-pane](https://user-images.githubusercontent.com/16223739/32839198-5258e734-ca1c-11e7-91a8-ba42ab71a2a5.png)

I checked quite a lot of pages and could not find really an example for a status message outside the pane. Can you confirm it they are always inside the pane? If that's the case I can remove the border.

Reply:
Guys hold on - these are the warning/notification messages that get populated as the popup in the top right of CiviCRM. Because of the fact that these were created in various different ways there is some JS that grabs them from these divs and then shows them as a popup in the right hand top corner. Lets not break these please!! Don't change existing CiviCRM functionality!!

Reply:
i'm not sure i understand: we are adding a border for an element that we can't even find? @JonScreat @mukeshcompucorp can you help @mihaelmladenov here?

Reply:
see here:
![image](https://user-images.githubusercontent.com/2102899/32839766-66ae2866-ca0d-11e7-8505-cab9fe694483.png)
Thats the same element that Mihael is talking about

Reply:
The mockup is for the element i've highlighted in blue here
![image](https://user-images.githubusercontent.com/19689834/32840100-8c829774-ca0e-11e7-9224-3c26c5dbe052.png)

Reply:
>  @AkA84 @JonScreat the style guide extension is therefor wrong and we need to make it clear what developers should be following.

@jamienovick i don't know how it is wrong, i'm not even sure what element @mihaelmladenov is talking about . You are talking about the notification popup while @JonScreat is talking about a standard alert message

Reply:
@jamienovick @AkA84 @mihaelmladenov 

@jamienovick you are correct. this should not get changed when we do any styling.

@mihaelmladenov the style in this ticket should be applied only to warning message which does not get converted to such popup. you will get class .no-popup when a pop is not going to be converted in popup
you can confirm that on the page you are stlying

Reply:
The ticket talks about this item: 
![image](https://user-images.githubusercontent.com/19689834/32840941-06596e18-ca11-11e7-94b6-c23bcdb6d8b3.png)

So this is the only element I've been talking about.

My advice was on changing the colour to the alert yellow from here: https://projects.invisionapp.com/d/main#/console/12482552/261993810/preview
(see the items at the bottom of the page) 

But that we shouldn't have a border around it in this instance as it sits within the panel and that only ones that sit on the page background should have a border.
e.g. the element outlined in blue here: 
![image](https://user-images.githubusercontent.com/19689834/32841148-80b74338-ca11-11e7-97e8-9fdecd1e5849.png)

I've not spoken about functionality of any of the pop-ups in the corner - we have a style for these in the current styleguide, and we shouldn't be changing that. Just updating the colours of the text and icons to match the new ones.

Reply:
@jamienovick @JoeMurray @AkA84 

After a call with Alessandro, it became clear that the confusion came from miscommunication. So what I was talking about was always the yellow box on the Cancel Event Participants page. The screenshot I sent of the Manage Groups page was just an example, where we have the same wrapper element, but it gets turned into a popup. So by altering the styles for the yellow box, I am not altering at all the popups.

We agreed with Alessandro to change the background color and border of the status box to the new color palette globally and to remove the border of the status box only when it is inside a panel.

I guess this solves the confusion. Will try to be more careful in communicating from now on.

Reply:
ok @jamienovick @JonScreat turned that none of those elements are the subject of this PR here

---

@jamienovick the elements that are transformed to into a popup are elements that match this selector
```js
$('div.messages:visible').not('.help').not('.no-popup')
```
so elements that have the `.messages` class but neither the `.help` nor the `.no-popup` class

moreover, when the element is put in the notification, it gets stripped of the `.status` and `.messages` classes
```js
$(this).removeClass('status messages');
```
@mihaelmladenov here is styling the `.status` class, so the notification popup will not be affected

---

@JonScreat the green alert/message here
![help](https://user-images.githubusercontent.com/19689834/32841148-80b74338-ca11-11e7-97e8-9fdecd1e5849.png)
it's *not* the same as this alert/message here
![status](https://user-images.githubusercontent.com/19689834/32840941-06596e18-ca11-11e7-94b6-c23bcdb6d8b3.png)
but @mihaelmladenov did not catch that, so you were basically talking about two different elements without realizing it

The second element cannot appear directly on the background, as that's the kind of element that will be grabbed by JS and transformed into a popup notification (see above)